### DISCUSSION TITLE
not able to train RE model on funsd dataset(english)

### DISCUSSION THREAD

**Discussion by utkarsh-tyagi** [PaddlePaddle/PaddleOCR#12857] [Q&A]
Hi Team , while training the RE model on funsd dataset , I am getting below error : 

Training command : # train RE model using single card
python3 tools/train.py -c configs/kie/vi_layoutxlm/re_vi_layoutxlm_xfund_zh.yml

**TypeError: LayoutXLMForRelationExtraction.__init__() got an unexpected keyword argument 'dropout'**

How to resolve below error?

**Comment by IbrarBabar009** [comment]
is your problem solved?

**Comment by tran601** [comment]
same problem
